praw/models/reddit/subreddit.py

```diff
@@ -2116,7 +2116,7 @@ def invite(self, redditor, permissions=None, **other_settings):
             provided ``None``, indicates full permissions.
 
         For example, to invite ``'spez'`` with ``'posts'`` and ``'mail'``
-            permissions to ``'/r/test/``, try:
+            permissions to ``'r/test/``, try:
```

Comment:
Looks like the `'` probably shouldn't be there, nor do we need the trailing `/`.